app/pages/ServiceListingPage.tsx

```diff
@@ -132,22 +143,3 @@ export const ServiceProgramDetails = ({ service, organization }: ServiceProgramD
     <ListingTitleLink type="org" listing={organization} />
   </p>
 );
-
-// Shows a section with relevant service fields in a table
-export const ServiceDetailsTableSection = ({ service }: { service: Service }) => {
```

Comment:
Only really added this to abstract the null render, but given the hidden convention on sections, seemed redundant.